trillian/ctfe/ct_server/main.go

```diff
@@ -113,23 +125,36 @@ func main() {
 		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
 		res = util.FixedBackendResolver{}
 	}
-	bal := grpc.RoundRobin(res)
-	conn, err := grpc.Dial(*rpcBackend, grpc.WithInsecure(), grpc.WithBlock(), grpc.WithBalancer(bal))
-	if err != nil {
-		glog.Exitf("Could not connect to rpc server: %v", err)
+
+	// Dial all our log backends.
+	clientMap := make(map[string]trillian.TrillianLogClient)
+	for _, be := range beMap {
+		glog.Infof("Dialling backend: %v", be)
+		var conn *grpc.ClientConn
```

Comment:
Isn't this now unnecessary because of the `conn:=` below?

Reply:
Yes, removed.